### PR TITLE
Bootstrap: Toast: Add isShown() method

### PR DESCRIPTION
**Pull request by luc122c**
This PR adds the `isShown()` method [documented here](https://getbootstrap.com/docs/5.2/components/toasts/#methods). 

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:
If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: [https://getbootstrap.com/docs/5.2/components/toasts/#methods](https://getbootstrap.com/docs/5.2/components/toasts/#methods)
- [ ] ~~If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.~~

### CODE REVIEW

File: types/bootstrap/test/toast-tests.ts [hunk 1]
```diff
@@ -10,6 +10,8 @@ new Toast(element, { animation: false });
 Toast.getInstance(element);
 // $ExpectType Toast
 Toast.getOrCreateInstance(element);
+// $ExpectType boolean
+Toast.isShown();
```
Comment (peterblazejewicz) [inline types/bootstrap/test/toast-tests.ts:14]: isn't that one an instance method?
https://github.com/twbs/bootstrap/blob/main/js/src/toast.js#L133

Comment (luc122c) [reply types/bootstrap/test/toast-tests.ts:14]: Yes, however I'm not sure how to express this as a test. In my code I'd use it like this:
```javascript
const myToast = new Toast(element, { animation: false });
const myBool = myToast.isShown(); 
```

Comment (martin-badin) [reply types/bootstrap/test/toast-tests.ts:14]: You can replace line 7 with your code.

Comment (luc122c) [reply types/bootstrap/test/toast-tests.ts]: Thank you for your help @peterblazejewicz @martin-badin :)

File: types/bootstrap/test/toast-tests.ts [hunk 2]
```diff
@@ -4,7 +4,8 @@ import * as $ from 'jquery';
 const element = new Element();
 
 // $ExpectType Toast
-new Toast(element, { animation: false });
+const myToast = new Toast(element, { animation: false });
```
Comment (martin-badin) [inline types/bootstrap/test/toast-tests.ts:7]:
```suggestion
// $ExpectType Toast
const toast = new Toast(element, { animation: false });
```

File: types/bootstrap/test/toast-tests.ts [hunk 3]
```diff
@@ -4,7 +4,8 @@ import * as $ from 'jquery';
...
+const myBool = myToast.isShown();
```
Comment (martin-badin) [inline types/bootstrap/test/toast-tests.ts:8]:
```suggestion
// $ExpectType boolean
toast.isShown();
```

**Review by martin-badin — CHANGES_REQUESTED** [review]

**Review by martin-badin — APPROVED** [review]

**Comment by luc122c** [comment]
Ready to merge

**Pull request merged**